DEVELOPER_GUIDE.md

```diff
@@ -31,7 +32,7 @@ Edit your Markdown file and insert the following snippet where you want render a
 <!-- spec_insert_start 
 api: <API_NAME>
 component: <COMPONENT_NAME>
-other_param: <OTHER_PARAM>
+other_argument: <OTHER_ARGUMENT>
```

Comment:
Calling this argument to match what the code is actually using and avoid confusion with the API path and query params.